jobs/openshift4/cluster/create/openshift4-cluster-create.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- job:
+    name: openshift4-cluster-create
+    display-name: 'OpenShift 4 Cluster Create'
+    project-type: pipeline
+    concurrent: true
+    parameters:
+      - string:
+          name: 'CLUSTER_NAME'
+          default: ''
+          description: 'Name of the cluster (e.g. your kerberos username). Defaults to build number'
+      - choice:
+          name: 'COMPUTE_REPLICAS'
+          choices:
+            - '2'
+            - '3'
+          description: 'Number of compute replicas for the Openshift 4 cluster'
+          default: '2'
+      - choice:
+          name: 'CONTROL_PLANE_REPLICAS'
+          choices:
+            - '1'
+            - '2'
+            - '3'
+          description: 'Number of control plane replicas for the Openshift 4 cluster'
+          default: '1'
+      - string:
+          name: 'RECIPIENTS'
+          default: integreatly-qe@redhat.com
+          description: "Whitespace- or comma-separated list of email addresses to which the result of this pipeline execution will be sent"
+      - password:
+          name: 'AWS_ACCESS_KEY_ID'
+          default: ''
+          description: 'AWS Access Key ID used to access AWS CLI'
+      - password:
+          name: 'AWS_SECRET_ACCESS_KEY'
+          default: ''
+          description: 'AWS Secret Access Key used to access AWS CLI'
```

Comment:
We need to lock this job down to one account initially. I'm not sure if merging this with the option to use any AWS credentials is th ebest idea since we have no way of knowing what account it is goign to be created in, and therefore what account it should be destroyed in after.

Reply:
ok. I will create two more secrets, which then can be updated if required, and hardcode them into the pipeline definition